### PR TITLE
Translate Fukuoka Ruby Award post (pt)

### PR DESCRIPTION
**Pull request by vnbrs**

### CODE REVIEW

File: pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: "Competição Fukuoka Ruby Award 2019 - Aplicações a serem julgadas pelo Matz"
+author: "Fukuoka Ruby"
+translator: "vnbrs (Vinicius Brasil)"
+date: 2018-11-29 00:00:00 +0000
+lang: pt
+---
+
+Caros entusiatas de Ruby,
+
+O governo de Fukuoka, Japão, juntamente com "Matz" Matsumoto gostaria de convidá-los para entrar na seguinte competição de Ruby. Se você desenvolveu algum programa interessante em Ruby, por favor seja encorajado à aplicar.
```
Comment (jcserracampos) [inline pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
O governo de Fukuoka, Japão, juntamente com "Matz" Matsumoto gostaria de convidá-los para entrar na seguinte competição de Ruby. Se você desenvolveu algum programa interessante em Ruby, por favor seja encorajado a aplicar.
```

Comment (jcserracampos) [reply pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]: You do not use crass before a verb.

File: pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Competição Fukuoka Ruby Award 2019 - Grande Prêmio - 1 Milhão de Ienes!
+
+Data máxima para aplicação: 31 de Janeiro de 2019
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz e um grupo de jurados selecionarão os vencedoras da competição de Fukuoka. O grande prêmio para essa competição é 1 milhão de ienes. Fora o grande prêmio, outras premiações incluem Rhomobile (EUA) and APEC Climate Center (Coréia).
```
Comment (jcserracampos) [inline pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
Matz e um grupo de jurados selecionarão os vencedores da competição de Fukuoka. O grande prêmio para essa competição é 1 milhão de ienes. Fora o grande prêmio, outras premiações incluem Rhomobile (EUA) and APEC Climate Center (Coréia).
```

Comment (jcserracampos) [inline pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
Matz e um grupo de jurados selecionarão os vencedoras da competição de Fukuoka. O grande prêmio para essa competição é 1 milhão de ienes. Fora o grande prêmio, outras premiações incluem Rhomobile (EUA) e APEC Climate Center (Coréia do Sul).
```

File: pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Esse ano, temos alguns prêmios especiais:
+
+O ganhador do prêmio AWS receberá:
+
+* Amazon Echo (sujeito à mudança)
+* Consultoria técnica de arquitetos AWS
+
+O ganhador do prêmio GMO Pepabo receberá:
+
+* Lolipop! (serviço de hospedagem compartilhada): assinatura gratuita por 10 anos do plano Standard, ou um cupom no valor de JPY 100,000 para o plano metrificado da Managed Cloud
```
Comment (jcserracampos) [inline pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
* Lolipop! (serviço de hospedagem compartilhada): assinatura gratuita por 10 anos do plano Standard, ou um cupom no valor de ¥ 100.000,00 para o plano metrificado da Managed Cloud
```

File: pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* Muumuu Domain (serviço de registro de DNS): assinatura gratuita por 10 anos de um domínio (válido somente para domínios de JPY 10,000 ou menos por ano)
```
Comment (jcserracampos) [inline pt/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
* Muumuu Domain (serviço de registro de DNS): assinatura gratuita por 10 anos de um domínio (válido somente para domínios de ¥ 10.000,00 ou menos por ano)
```

**Comment by vnbrs** [comment]
:shipit:

**Comment by JuanitoFatas** [comment]
This is great, thanks folks!

**Pull request merged**